src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
Some fields have been obsoleted in ClusterCreateParameters; making this a breaking change for all users of the SDK. Hence the major version bump.

src\resourcemanagement\hdinsight\hdinsight\customizationmodels\clustercreateparameters.cs:

     [ObsoleteAttribute("This property is deprecated. Please use AzureStorageInfo with the DefaultStorageInfo parameter instead.", true)]
        public string DefaultStorageAccountName { get; set; }

        /// <summary>
        /// Gets or sets the StorageKey for the default Azure Storage Account.
        /// This account will be used for schemaless paths and the cluster will 
        /// leverage to store some cluster level files.
        /// </summary>
        [ObsoleteAttribute("This property is deprecated. Please use AzureStorageInfo with the DefaultStorageInfo parameter instead.", true)]
        public string DefaultStorageAccountKey { get; set; }

        /// <summary>
        /// Gets or sets the StorageContainer for the default Azure Storage Account.
        /// This account will be used for schemaless paths and the cluster will 
        /// leverage to store some cluster level files.
        /// </summary>
        [ObsoleteAttribute("This property is deprecated. Please use AzureStorageInfo with the DefaultStorageInfo parameter instead.", true)]
        public string DefaultStorageContainer { get; set; }